desc/objectives/_stability.py

```diff
@@ -407,12 +405,12 @@ def __init__(
         normalize_target=True,
         loss_function=None,
         deriv_mode="auto",
-        rho=0.5,
+        rho=np.array([0.5]),
         alpha=None,
         nturns=3,
         nzetaperturn=200,
-        zeta0=None,
-        Neigvals=None,
+        zeta0=jnp.linspace(-0.5 * np.pi, 0.5 * np.pi, 15),
```

Comment:
note that making arrays default args sometimes messes with autodoc stuff.

FYI `BallooningStability` is also missing from the docs